ref_ts/static/ref_ts/rickshaw/.jshintrc

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
@zhiyuli Same with rickshaw.  @mauriel133 @horsburgh How would you prefer we handle JS dependencies?  My gut tells me we should minimize the number of JS libraries that we allow resource instances to import, and try to consolidate JS libraries, as much as possible, within hs_core.